.github/workflows/build.yml

```diff
@@ -199,6 +224,17 @@ jobs:
       matrix: ${{toJson(matrix.matrix_override)}}
       python_deps_ids: ${{toJson(matrix.python_deps_ids)}}
       persistent_storage: ${{inputs.persistent_storage}}
+
+  stop-ec2-runner:
+    needs: [start_linux_ec2_runner, follower-linux]
```

Comment:
You only need to wait until `leader-compile-linux` to finish, isn't it?

Also, if the user wants to rerun just the linux compile leader, it will never work because the server has been stopped